frontend/src/components/patient/CreatePatientForm.js

```diff
@@ -330,7 +336,9 @@ function CreatePatientForm(props) {
           days: "",
         });
       },
-    );
+    ).then(() => {
+      setIsSubmitting(false);
```

Comment:
you can set this directly in the call back [here](https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/develop/frontend/src/components/patient/CreatePatientForm.js#L338). when the call has been succesful